### DISCUSSION TITLE
Where Can I download GIAnT

### DISCUSSION THREAD

**Discussion by Arctic-Ambrun** [isce-framework/isce2#502] [Q&A] [answered]
Hello everyone, I want to download GIAnT to learn the performance of InSARFlow.
, however the link lead me to here.

Can any one tell me how to download GIAnT?
Please, THanks.

**Comment by fdelgadodelapuente** [comment, marked as answer]
GiANT is deprecated and no longer used nor updated. Switch to MintPy

**Comment by Arctic-Ambrun** [comment]
😂Thank you Mr.Francisco Delgado
________________________________
发件人: Francisco Delgado ***@***.***>
发送时间: 2022年6月14日 3:25
收件人: isce-framework/isce2 ***@***.***>
抄送: WANG, JiaGeng [Alumni] ***@***.***>; Author ***@***.***>
主题: Re: [isce-framework/isce2] Where Can I download GIAnT (Discussion #502)


GiANT is deprecated and no longer used nor updated. Switch to MintPy

—
Reply to this email directly, view it on GitHub<https://github.com/isce-framework/isce2/discussions/502#discussioncomment-2941338>, or unsubscribe<https://github.com/notifications/unsubscribe-auth/ATPQPAD7GJIOGIUCM36SSB3VO6DI7ANCNFSM5YTSNLBQ>.
You are receiving this because you authored the thread.Message ID: ***@***.***>

[https://www.polyu.edu.hk/emaildisclaimer/85A-PolyU_Email_Signature.jpg]

Disclaimer:

This message (including any attachments) contains confidential information intended for a specific individual and purpose. If you are not the intended recipient, you should delete this message and notify the sender and The Hong Kong Polytechnic University (the University) immediately. Any disclosure, copying, or distribution of this message, or the taking of any action based on it, is strictly prohibited and may be unlawful.

The University specifically denies any responsibility for the accuracy or quality of information obtained through University E-mail Facilities. Any views and opinions expressed are only those of the author(s) and do not necessarily represent those of the University and the University accepts no liability whatsoever for any losses or damages incurred or caused to any party as a result of the use of such information.
